extra/exports

```diff
@@ -356,3 +356,22 @@ luaJIT_setmode
 luaJIT_profile_start
 luaJIT_profile_stop
 luaJIT_profile_dumpstack
+
+ERR_error_string
+ERR_get_error
+
+EVP_get_digestbyname
+EVP_get_cipherbyname
+EVP_CIPHER_CTX_new
+EVP_CIPHER_CTX_free
+EVP_CIPHER_CTX_cleanup
+EVP_CIPHER_block_size
+EVP_CipherInit_ex
+EVP_CipherUpdate
+EVP_CipherFinal_ex
+EVP_DigestInit_ex
+EVP_DigestUpdate
+EVP_DigestFinal_ex
+HMAC_Init_ex
+HMAC_Update
+HMAC_Final
```

Comment:
It is not part of tarantool. Should not be here in a regular build. Please, consider updating of extra/mkexports.